### PR TITLE
coq-prosa.dev: update bounds

### PR DESCRIPTION
**Pull request by proux01**
According to upstream CI: https://gitlab.mpi-sws.org/RT-PROOFS/rt-proofs/-/pipelines/58088

### CODE REVIEW

File: extra-dev/packages/coq-prosa/coq-prosa.dev/opam [hunk 1]
```diff
@@ -12,8 +12,8 @@ build: [
 ]
 install: [make "install"]
 depends: [
-  "coq" {((>= "8.11" & < "8.13~") | = "dev")}
-  "coq-mathcomp-ssreflect" {((>= "1.10" & < "1.12~") | = "dev")}
+  "coq" {(>= "8.13" | = "dev")}
```
Comment (palmskog) [inline extra-dev/packages/coq-prosa/coq-prosa.dev/opam:15]:
```suggestion
  "coq" {>= "8.13"}
```

File: extra-dev/packages/coq-prosa/coq-prosa.dev/opam [hunk 2]
```diff
@@ -12,8 +12,8 @@ build: [
...
+  "coq-mathcomp-ssreflect" {(>= "1.12" | = "dev")}
```
Comment (palmskog) [inline extra-dev/packages/coq-prosa/coq-prosa.dev/opam:16]:
```suggestion
  "coq-mathcomp-ssreflect" {>= "1.12"}
```
Due to how opam works, `>= "1.12"` by itself already includes `dev`.

**Review by palmskog — COMMENTED** [review]
It looks like Prosa is not currently compatible with Coq master, is this expected? Since this is an extra-dev project, I can still merge the PR if you want me to @proux01.

**Comment by proux01** [comment]
Indeed, it's in allow_failure mode on upstream CI, I'll investigate upstream.
Meanwhile, I think this can be merged.

**Comment by proux01** [comment]
Thanks @palmskog !

**Pull request merged**